records/scrum_team_charter.md

```diff
@@ -0,0 +1,87 @@
+# Visual Meaning Scrum Team Charter
+
+## Things we always do
+
+- Try to write the best code
+
+- **Be kind**
+  - Assume best intent

```

Comment:
I prefer to ask before assume anything. I'm QA part of our training is to NOT ASSUME. Is this dev team charter or some company things? Observations are my job.